modules/infrastructure-node-sizing.adoc

```diff
@@ -37,9 +37,9 @@ In general, three infrastructure nodes are recommended per cluster.
 
 [IMPORTANT]
 ====
-These sizing recommendations are based on scale tests, which create a large number of objects across the cluster. These tests include reaching some of the cluster maximums. In the case of 250 and 500 node counts on an {product-title} {product-version} cluster, these maximums are 10000 namespaces with 61000 pods, 10000 deployments, 181000 secrets, 400 config maps, and so on. Prometheus is a highly memory intensive application; the resource usage depends on various factors including the number of nodes, objects, the Prometheus metrics scraping interval, metrics or time series, and the age of the cluster. The disk size also depends on the retention period. You must take these factors into consideration and size them accordingly.
+These sizing recommendations should be used as a guideline and not as a rule of thumb, the number and type of objects in a cluster can vary regardless of the number of nodes this cluster has. Prometheus is a highly memory intensive application; the resource usage depends on various factors including the number of nodes, objects, the Prometheus metrics scraping interval, metrics or time series, and the age of the cluster. In addition, the router component usage can also be affected by the amount of inbound requests and the type of them, it's important to note that the memory usage can also be affected by the number of routes the router is managing.
```

Comment:
```suggestion
These sizing recommendations should be used as a guideline.  Applications that are highly memory intensive can require more resources than prescribed in this chart. Memory requirements are increased by the use of Prometheus or the number of managed routes.
```

Reply:
I fixed some of the comments on this section, not all of the though. I understand the point of reducing this note but some I think we should keep some of the original content.